### ISSUE TITLE
[QA] Glasses (Transparent) wearables won't show

### ISSUE THREAD

**Issue by NickKhalow** [decentraland/unity-explorer#862]
<img width="1728" alt="image" src="https://github.com/decentraland/unity-explorer/assets/71646502/33fe0046-71cd-4c9d-8ffd-79a453475bf2">

- Put on your default glasses
- Compare with the old renderer

**Comment by m3taphysics** [comment]
Is this due to missing asset bundles @lorux0

**Comment by lorux0** [comment]
Yes it is because asset bundles. I can confirm that they are working again:

![Image](https://github.com/decentraland/unity-explorer/assets/56365551/32d705b1-8444-4618-a731-b6d671715022)

**Comment by anicalbano** [comment]
@lorux0 the glasses are unequipping my face (it's a helmet) which wasn't happening before: 
![image](https://github.com/decentraland/unity-explorer/assets/139157886/fbc7f398-b21b-4688-b39e-6ecbd048154a)

**Comment by lorux0** [comment]
> [@lorux0](https://github.com/lorux0) the glasses are unequipping my face (it's a helmet) which wasn't happening before: ![image](https://private-user-images.githubusercontent.com/139157886/328162056-fbc7f398-b21b-4688-b39e-6ecbd048154a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ5OTk3NDksIm5iZiI6MTcxNDk5OTQ0OSwicGF0aCI6Ii8xMzkxNTc4ODYvMzI4MTYyMDU2LWZiYzdmMzk4LWIyMWItNDY4OC1iMzllLTZlY2JkMDQ4MTU0YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwNlQxMjQ0MDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MjUwMzAzNjk5MGIxYjUwMzY0ZmRmZmI1ZGQ3ZDcxNTFjNDVjZjYzNDQ3MGZhYzEzNTRiMDNlZjUwMDZmNDQ2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.N82dUhCifjIqWu79-JfhTMsGJOG0Z1e6WLikMFuTHHc)

This was an asset bundle conversion problem. There are no changes in the client. Has to be something else causing this problem
